src/lib/wasm/RaygunStore.ts

```diff
@@ -110,6 +110,8 @@ class RaygunStore {
                 this.initConversationHandlers(r)
                 // handleRaygunEvent must be called after initConversationHandlers
                 // this is because if 'raygun.raygun_subscribe' is called before 'raygun.list_conversations', it causes 'raygun.list_conversations' to hang. (reason currently unknown)
+                // update: a delay is (also?) needed as otherwise raygun events are not received
+                await new Promise(f => setTimeout(f, 100))
```

Comment:
I get this is a workaround, but if possible I'd avoid adding arbitrary delays like this. we can't guarantee that whatever needs to happen will within 100ms. And setting a large value would cause unnecessary delays.

Generally, it would be better to have the prerequisite action complete and then fire an event or something that point which would trigger the next line to run.

Reply:
i just rewrote the logic here to instead retry again after a while if its hanging